.all-contributorsrc

```diff
@@ -0,0 +1,285 @@
+{
+  "projectName": "kottans.github.io",
+  "projectOwner": "kottans",
+  "repoType": "github",
+  "repoHost": "https://github.com",
+  "files": [
+    "README.md"
+  ],
+  "imageSize": 100,
+  "commit": true,
+  "commitConvention": "none",
+  "contributors": [
+    {
+      "login": "programulya",
+      "name": "Julia Savinkova",
+      "avatar_url": "https://avatars0.githubusercontent.com/u/2372431?v=4",
+      "profile": "http://www.programulya.com",
+      "contributions": [
+        "code"
+      ]
+    },
+    {
+      "login": "sudodoki",
+      "name": "Джон, просто Джон",
+      "avatar_url": "https://avatars3.githubusercontent.com/u/1506905?v=4",
+      "profile": "https://github.com/sudodoki",
+      "contributions": [
+        "code"
+      ]
+    },
+    {
+      "login": "suchov",
+      "name": "Artem Sychov",
+      "avatar_url": "https://avatars3.githubusercontent.com/u/1478985?v=4",
+      "profile": "http://artemsychov.com/",
+      "contributions": [
+        "code"
+      ]
+    },
+    {
+      "login": "mayosh",
+      "name": "Julia",
+      "avatar_url": "https://avatars0.githubusercontent.com/u/3317247?v=4",
+      "profile": "https://github.com/mayosh",
+      "contributions": [
+        "code"
+      ]
+    },
+    {
```

Comment:
not sure you want gitter-badger - this one was bot putting links to gitter chats into the repos (that rooms were created for)

Reply:
good spot, thanks!